orchagent/bufferorch.cpp

```diff
@@ -1001,8 +1018,17 @@ task_process_status BufferOrch::processQueue(KeyOpFieldsValuesTuple &tuple)
         // set order is detected.
         for (const auto &port_name : port_names)
         {
-            if (gPortsOrch->isPortAdminUp(port_name)) {
-                SWSS_LOG_WARN("Queue profile '%s' applied after port %s is up", key.c_str(), port_name.c_str());
+            if(local_port == true)
+            {
+                if (gPortsOrch->isPortAdminUp(local_port_name)) {
+                    SWSS_LOG_WARN("Queue profile '%s' applied after port %s is up", key.c_str(), port_name.c_str());
```

Comment:
should this be changed to use local_port_name.c_str() instead of port_name.c_str()?
Of for local_port is true case, port_name is always the same as local_port_name?

Reply:
i would like to keep port_name in the debug message since local_port_name will be subset of port_name.